schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
 	date_of_birth date,
 	escape_attempts  integer,
 	neutered  boolean,
-	weight_kg  decimal
+	weight_kg  decimal,
+	species varchar(50)
 );
```

Comment:
- [ ] Instead of manually adding the `species` column to the `animals` table kindly use the `ALTER TABLE` command. The `SQL ALTER TABLE` command is used to modify a table's structure by adding, or deleting columns in an existing table. You can also use the `ALTER TABLE `command to add and drop various constraints on an existing table.

#### Hint: `ALTER TABLE animals ADD species varchar(255);`

Reply:
@Gambit142 Thanks for this suggestion but as you can see from the screenshot I did add species using the ALTER TABLE method, I might have cleared the piece of code but the value was inserted into the table.

![Screenshot (188)](https://github.com/AleWaweru/vet-clinic-database/assets/102799430/4288fd04-7555-4194-ae38-f72b4e02fd9f)

Reply:
Hi @AleWaweru, I am sorry, but you manually added it to your code. This is the code you have in the _**schema.sql**_ file:

https://github.com/AleWaweru/vet-clinic-database/blob/48a282fc803e531a4069afe77050179c60b6d620/schema.sql#L1-L11

#### As you can see, on line 10, you manually added the species column. Kindly use the ALTER TABLE command. 👍 😎